packages/core-bridge/src/worker.rs

```diff
@@ -498,6 +536,56 @@ mod config {
         }
     }
 
+    impl From<WorkerDeploymentOptions> for CoreWorkerDeploymentOptions {
+        fn from(val: WorkerDeploymentOptions) -> Self {
+            Self {
+                version: val.version.into(),
+                use_worker_versioning: val.use_worker_versioning,
+                default_versioning_behavior: Some(val.default_versioning_behavior.into()),
+            }
+        }
+    }
+
+    impl From<WorkerDeploymentVersion> for CoreWorkerDeploymentVersion {
+        fn from(val: WorkerDeploymentVersion) -> Self {
+            Self {
+                build_id: val.build_id,
+                deployment_name: val.deployment_name,
+            }
+        }
+    }
+
+    impl From<CoreWorkerDeploymentVersion> for WorkerDeploymentVersion {
+        fn from(val: CoreWorkerDeploymentVersion) -> Self {
+            Self {
+                build_id: val.build_id,
+                deployment_name: val.deployment_name,
+            }
+        }
+    }
+
+    impl TryIntoJs for WorkerDeploymentVersion {
+        type Output = JsObject;
+
+        fn try_into_js<'cx>(self, cx: &mut impl Context<'cx>) -> JsResult<'cx, Self::Output> {
```

Comment:
Seems like you can simply add `#derive(TryIntoJs)` on the `WorkerDeploymentVersion` struct. 

And if you really need to do it by hand, prefer using the `obj.set_property_from(cx, "buildId" seld.build_id)?;` syntax.

Reply:
No, you can't. I meant to message you about that - but, into and from don't appear to work together. Seems like something that needs to be fixed in the macros.

Reply:
Oh, then I'll revisit the macro at a later time.

For now, though, can still use the `set_property_from` macro instead, just to keep the code a bit cleaner.